src/Tasks/Common/Resources/xlf/Strings.cs.xlf

```diff
@@ -511,8 +511,8 @@ The following are names of parameters or literal values and should not be transl
         <note>{StrBegin="NETSDK1053: "}</note>
       </trans-unit>
       <trans-unit id="PackAsToolCannotSupportTargetPlatformIdentifier">
-        <source>NETSDK1146: Pack as tool does not support TargetPlatformIdentifier being set. For example, targetframework cannot be net5.0-windows, only net5.0. And Pack as tool .NET5 or higher does not support UseWPF and UseWinform.</source>
-        <target state="new">NETSDK1146: Pack as tool does not support TargetPlatformIdentifier being set. For example, targetframework cannot be net5.0-windows, only net5.0. And Pack as tool .NET5 or higher does not support UseWPF and UseWinform.</target>
+        <source>NETSDK1146: PackAsTool does not support TargetPlatformIdentifier being set. For example, TargetFramework cannot be net5.0-windows, only net5.0. PackAsTool also does not support UseWPF or UseWindowsForms when targeting .NET 5 and higher.</source>
+        <target state="new">NETSDK1146: PackAsTool does not support TargetPlatformIdentifier being set. For example, TargetFramework cannot be net5.0-windows, only net5.0. PackAsTool also does not support UseWPF or UseWindowsForms when targeting .NET 5 and higher.</target>
```

Comment:
Long term, would it make sense to have a {0} placeholder, then build a list of property names that cannot be supported in case there are more to be added in the future besides the 3 listed here now?